lightningd/closing_control.c

```diff
@@ -625,6 +657,17 @@ static struct command_result *json_close(struct command *cmd,
 		channel->shutdown_scriptpubkey[LOCAL]
 			= tal_steal(channel, cast_const(u8 *, close_to_script));
 		close_script_set = true;
+		/* Is the close script in our wallet? */
+		bool is_p2sh;
+		if (wallet_can_spend(
+			    cmd->ld->wallet,
+			    channel->shutdown_scriptpubkey[LOCAL],
+			    &index_val,
+			    &is_p2sh)) {
+			final_index = &index_val;
```

Comment:
this seems to be a noop, since it's already set to point to `index_val` above

Reply:
removed; replaced w/ a comment that points out that index_val was updated